### ISSUE TITLE
QUIC: Senders MUST either use pacing or limit such bursts.

### ISSUE THREAD

**Issue by nhorman** [openssl/project#900]
This is a missing MUST item from the spreadsheet here:
https://docs.google.com/spreadsheets/d/1is0eRNrmNwzqcCTmTPYJwC3fswpYpqmY87-5CylraLc/edit?gid=1067533579#gid=1067533579

Need to determine if this is still missing, and if it is, correct it

Investigate and, if work is required, create another issue with a description of what needs to be done.

**Comment by nhorman** [comment]
Note, this requirement is part of RFC 9002, not RFC 9000.  While important, I'm struggling to see that this is an MVP requirement for QUIC functionality

**Comment by t8m** [comment]
Yeah, this is a RFC 9002 violation but it applies equally to both client and server. As we did not implement it for the client I do not think this is a blocker for the server MVP either. That doesn't mean we shouldn't implement it rather sooner than later.

**Comment by nhorman** [comment]
Agreed, we should definitely do it, it can just be a lower priority than MVP requirements
